precise_description/robots/PF400_long.yaml

```diff
@@ -22,7 +22,7 @@ arm:
     limit:
       effort: 1000.0
       lower: 0.0015  # m
-      upper: 0.750  # m - 401,5mm standard, 750 mm, 1160 mm
+      upper: 0.730  # m - 401,5mm standard, 750 mm, 1160 mm
```

Comment:
@benmaidel @destogl
is this the official upper limit? otherwise, I'd rather change this in a mrk-specific yaml in mojin_hardware_config

Reply:
no, that's not the offical limit. It is the measured limit from the robot. I also think that we should have a mrk specific yaml. But structure is just not there yet and this was just a quick workaround.

Reply:
structure is in https://github.com/mojin-robotics/mojin_robots/pull/377